src/main.rs

```diff
@@ -119,9 +125,15 @@ fn list_open_requests() {
     let mut tw = TabWriter::new(io::stdout()).padding(4);
     for mr in &mrs {
         if remote.has_useful_branch_names() {
-            writeln!(&mut tw, "{}\t{}\t{}", mr.id, mr.source_branch, mr.title).unwrap();
+            writeln!(
+                &mut tw,
+                "{}", format!("{}\t{}\t{}", mr.id, mr.source_branch, mr.title).green()
```

Comment:
It'd be easier to visually parse this output if we apply different colors. I'm thinking that: `mr.id` is the most important, so it should stand out.  Maybe `green` for the `mr.id`, `dimmed` for the `source_branch` and uncolored for the `title`?  I can worry more about tweaking these in future versions.

Reply:
Done.